src/transformers/generation/configuration_utils.py

```diff
@@ -306,6 +309,7 @@ def __init__(self, **kwargs):
         self.output_attentions = kwargs.pop("output_attentions", False)
         self.output_hidden_states = kwargs.pop("output_hidden_states", False)
         self.output_scores = kwargs.pop("output_scores", False)
+        self.output_logits = kwargs.pop("output_logits", False)
```

Comment:
```suggestion
        self.output_logits = kwargs.pop("output_logits", None)
```
We'd rather default to unset (`None`) than to a valid option (`False`) on all new attributes. That way, if we want to change the default in the future, we can easily know when a user has intentionally set this attribute :)